CRM/Bsd/Page/BSD.php

```diff
@@ -36,6 +36,14 @@ class CRM_Bsd_Page_BSD extends CRM_Core_Page {
 
   public $new_contact = false;
 
+  private $apiAddressGet = 'api.Address.get';
```

Comment:
I moved local variable `$apiAddressGet` from method `createContact` because I devided this method into two new methods: `createContact` and `prepareParamsContact`. I want to have one definition, not two